src/log-tlslog.c

```diff
@@ -294,8 +300,10 @@ static int LogTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
         return 0;
     }
 
-    if (ssl_state->server_connp.cert0_issuerdn == NULL ||
-            ssl_state->server_connp.cert0_subject == NULL) {
+    if ((ssl_state->server_connp.cert0_issuerdn == NULL ||
+            ssl_state->server_connp.cert0_subject == NULL) &&
+            ((ssl_state->flags & SSL_AL_FLAG_SESSION_RESUMED) == 0 ||
+            (hlog->flags & LOG_TLS_SESSION_RESUMPTION) == 0)) {
```

Comment:
micro optimization: I wonder if we should check for hlog->flags & LOG_TLS_SESSION_RESUMPTION first? Safes a lot of ssl_state access if disabled, and since it's a more or less global variable it probably more likely to be hot in cache.